actions/routeActions.js

```diff
@@ -1,6 +1,38 @@
-import { loadPropertyListings } from './init';
+// Actions
+import { loadPropertyListings, getCoordinates } from './init';
+
+// Utils
+import { executeMultiple } from 'fluxible-action-utils/async';
+
+// Stores
+import LocationStore from '../stores/LocationStore';
 
 export const getListings = (context, route, done) => {
     route = route || {};
-    context.executeAction(loadPropertyListings, {}, done);
-};
\ No newline at end of file
+
+    const actions = {
+    	loadLocation: {
+    		action: getCoordinates,
+    		isCritical: true
```

Comment:
isn't this defaulted to 'true'?

Reply:
no defaulted to false. this will allow us to add 403 if this fails. though i dont think its necessary anymore, should just display search bar with random listings if this fails